Deceive/Utils.cs

```diff
@@ -358,10 +358,18 @@ public static WebSocket MonitorChatStatusChange(string status)
                 ws.OnMessage += (s, e) =>
                 {
                     if (!e.IsText) return;
+                    var json = (JsonArray) SimpleJson.DeserializeObject(e.Data);
+                    if ((long) json[0] != 8) return;
+                    var statusJson = (JsonObject)((JsonObject) json[2])[0];
+                    if (!statusJson.ContainsKey("availability")) return;
+                    var availability = (string) statusJson["availability"]; 
+                    if (availability == "dnd" || availability == status || availability == "away") return;
+                    Trace.WriteLine((string) statusJson["availability"]);
                     SendStatusToLcu(status);
+                    if (!enabled) ws.Close();
                 };
                 ws.Connect();
-                ws.Send("[5, \"OnJsonApiEvent_lol-gameflow_v1_gameflow-phase\"]");
+                ws.Send("[5, \"OnJsonApiEvent_lol-chat_v1_me\"]");
```

Comment:
Back to looking for changes in `lol-chat/v1/me`, but this time just calling SendStatusToLcu() when we actually can and need.